src/pages/SingleArticle.jsx

```diff
@@ -1,4 +1,32 @@
-// function SingleArticle() {
+import { useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
+import { getArticle } from "../utils/api";
+import "../styling/Articles.css";
 
-// }
-// export default SingleArticle;
\ No newline at end of file
+function SingleArticle({ articles }) {
```

Comment:
Why are you passing articles down on props? I don't think it would be needed in this component and it doesn't look like you're using it!